tests/slsa_analyzer/package_registry/test_npm_registry.py

```diff
@@ -0,0 +1,131 @@
+# Copyright (c) 2023 - 2023, Oracle and/or its affiliates. All rights reserved.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/.
+
+"""Tests for the npm registry."""
+
+import os
+from pathlib import Path
+
+import pytest
+
+from macaron.config.defaults import load_defaults
+from macaron.errors import ConfigurationError
+from macaron.slsa_analyzer.build_tool.base_build_tool import BaseBuildTool
+from macaron.slsa_analyzer.package_registry.npm_registry import NPMAttestationAsset, NPMRegistry
+
+
+def _create_npm_registry() -> NPMRegistry:
+    """Create an npm registry instance.
+
+    This private function is used as parameter for Hypothesis, which otherwise does not accept a fixture.
```

Comment:
Hmm, I don't see this being used in any hypothesis tests.

Reply:
Yes, forgot to remove it after moving the hypothesis test.